services/web-app/data/licenses.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const licenses = {
+  'apache-2.0': {
+    name: 'Apache 2.0'
+  },
+  'ibm-internal': {
+    name: 'IBM Internal'
```

Comment:
We might want this to be "IBM proprietary" per our OSS manager, but I'll create a separate issue to track this. Not urgent to do now.

Reply:
Instead of "IBM internal".